docs/developer/setup_develop_environment.rst

```diff
@@ -33,7 +21,7 @@ command below.
 
 .. code-block:: shell
 
-  $ sudo apt install git libpython3.6-dev python3.6 python3-venv
+  $ apt install git python3 python3-venv libpython3-dev
```

Comment:
The same here.
```suggestion
  $ sudo apt install git python3 python3-venv libpython3-dev
```

Reply:
As I said, `sudo apt` doesn't seem to be required nowadays.